com.unity.ml-agents/Runtime/Sensors/ISensor.cs

```diff
@@ -16,6 +16,28 @@ public enum SensorCompressionType
         PNG
     }
 
+    /// <summary>
```

Comment:
Whether or not this enum makes it into master will ultimately be up to the toolkit pod but for our purposes whatever works.

Reply:
Agreed. This is just for testing purposes.